exchanges/stream/websocket_connection.go

```diff
@@ -298,6 +298,10 @@ func (w *WebsocketConnection) SendMessageReturnResponses(ctx context.Context, si
 		return nil, err
 	}
 
+	if request.IsVerbose(ctx, w.Verbose) {
+		log.Debugf(log.WebsocketMgr, "%v %v websocket connection: Sending message: %v", w.ExchangeName, w.URL, string(outbound))
+	}
+
 	start := time.Now()
 	err = w.SendRawMessage(ctx, websocket.TextMessage, outbound)
```

Comment:
This will double log. The only allowable double-log is in the unflushed company toilet

Reply:
```
[DEBUG] | WEBSOCKET | 05/09/2024 14:05:39 | Kucoin wss://ws-api-spot.kucoin.com/?token=POPCAT websocket connection: Sending message: {"id":"167756407","type":"subscribe","topic":"/account/balance","privateChannel":true,"response":true}
...
[DEBUG] | WEBSOCKET | 05/09/2024 14:05:40 | Kucoin websocket connection: sending message [{"id":"167756407","type":"subscribe","topic":"/account/balance","privateChannel":true,"response":true}]
[DEBUG] | WEBSOCKET | 05/09/2024 14:05:40 | Kucoin websocket connection: message received: {"id":"167756407","type":"ack"}
[DEBUG] | WEBSOCKET | 05/09/2024 14:05:40 | Kucoin wss://ws-api-spot.kucoin.com/?token=POPCAT websocket connection: Received response [1 out of 1]: {"id":"167756407","type":"ack"}

```

Reply:
I know its verbose, but I don't think the full URL is a good addition, especially with kucoin where the token is actually 218 characters long

Reply:
I kept the URL and just removed queries from the string [here](https://github.com/thrasher-corp/gocryptotrader/pull/1640/commits/8ef13ebab289a9b8ff2fbfcff9cd479f6accf92a). The distinction can be helpful when there are multiple different connections for each asset or auth type. Should be good to check over now.